.gitignore

```diff
@@ -15,3 +15,5 @@ icon-builder/jsx
 
 # OS files
 .DS_STORE
+
+.tern-port
```

Comment:
I believe that we `.tern-port` has nothing to do here. Why not adding it to your global `.gitignore`?
In the same idea. I would just kill the `# Exclude IDE project folders` section.